sdk/openai/openai/src/models/options.ts

```diff
@@ -233,4 +233,14 @@ export interface GetChatCompletionsOptions extends OperationOptions {
    *   This additional specification is only compatible with Azure OpenAI.
    */
   azureExtensionOptions?: AzureExtensionsOptions;
+  /**
+   * Whether to return log probabilities of the output tokens or not.If true, returns the log probabilities of
+   * each output token returned in the `content` of `message`. This option is currently not available on the `gpt-4-vision-preview` model.
+   */
+  logprobs?: boolean;
+  /**
+   * An integer between 0 and 5 specifying the number of most likely tokens to return at each token position,
+   * each with an associated log probability. `logprobs` must be set to `true` if this parameter is used.
+   */
+  topLogprobs?: number;
```

Comment:
I wonder do we need two knobs to control this feature? can't we just do with `topLogprobs`? Perhaps it is more of a question to the architects.

Reply:
This is a pattern in OpenAI API, so seems like most languages are surfacing the pattern. In .NET, they name `logprobs` as `EnableLogProbabilities` and `topLogprobs` as `LogProbabilitiesPerToken`.

Reply:
This is a pattern in OpenAI API, so seems like most languages are surfacing the pattern. In .NET, they rename `logprobs` as `EnableLogProbabilities` and `topLogprobs` as `LogProbabilitiesPerToken`.